internal/reader/onchain_prices_reader.go

```diff
@@ -84,5 +89,57 @@ func (pr *OnchainTokenPricesReader) GetTokenPricesUSD(
 	return prices, nil
 }
 
+func (pr *OnchainTokenPricesReader) getRawTokenPrice(ctx context.Context, token types.Account) (*big.Int, error) {
+	var latestRoundData *LatestRoundData
+	if err :=
+		pr.ContractReader.GetLatestValue(
+			ctx,
+			consts.ContractNamePriceAggregator,
+			consts.MethodNameGetLatestRoundData,
+			primitives.Unconfirmed,
+			nil,
+			&latestRoundData,
+			//boundContract,
```

Comment:
How simple would it be to fix this call once the breaking changes are in? We need the contract address to call. Seems to me it should be fine since we have all the addresses in the PriceSource field.

Reply:
Should be pretty straight forward, I even prepared the `boundContract` before the call as you can see.